### ISSUE TITLE
Cannot moveByPath with the result of a PathFinder.search

### ISSUE THREAD

**Issue by wesleyhall** [exaV/screeps-kotlin-types#70]
In the screeps API docs it is suggested that the argument to Creep.moveByPath can either be the result of Room.findPath in both plain and serialised form, or the result of PathFinder.search: 

https://docs.screeps.com/api/#Creep.moveByPath

It appears that the kotlin types do not support the last of these options. The GenericCreep interface provides: 

```kotlin
fun moveByPath(path: Array<Room.PathStep>): ScreepsReturnCode
fun moveByPath(serializedPath: String): ScreepsReturnCode
```

If the docs are correct this may just be a matter of introducing: 

```kotlin
fun moveByPath(path: PathFinder.Path): ScreepsReturnCode
```

**Comment by wesleyhall** [comment]
Ok, I did a quick check on the console in Javascript. The above is not quite right. moveByPath does not accept PathFinder.Path but it does accept the value in the .path property of PathFinder.Path which is Array<RoomPosition>, so I think the correct addition is... 

```kotlin
fun moveByPath(path: Array<RoomPosition>): ScreepsReturnCode
```

**Comment by exaV** [comment]
fixed in #71
